KVBench.py

```diff
@@ -187,24 +190,11 @@ def getSummaryList(threadid, phase):
 	return retdata
 
 
-
-def DataPrimeThread(threadid):
-	for i in range(2000):
-		StoreKV(threadid, "DataPrime")
-		# time.sleep(random.uniform(0.5, 1.5))
-
 def DataPrime(threadcount):
 	starttime = time.time()
 	print "Started DataPrime("+str(threadcount)+") "+time.strftime("%x %X %Z")
 	for i in range(threadcount):
-		t = threading.Thread(target=DataPrimeThread, args=([i]))
-		t.start()
-		threads.append(t)
-		# print "Started thread :"+str(i)
-
-	for t in threads:
-		if t.isAlive():
-			t.join()
+		StoreKV(i, "DataPrime", 2000)

```

Comment:
There isn't a problem using threads for the POST, I've just kept it on the main thread for the moment